ibm_catalog.json

```diff
@@ -48,6 +48,46 @@
                 }
               ]
             },
+            "dependency_version_2": true,
+            "dependencies": [
+              {
+                "flavors": [
+                  "fully-configurable"
+                ],
+                "catalog_id": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3",
+                "id": "045c1169-d15a-4046-ae81-aa3d3348421f-global",
+                "name": "deploy-arch-ibm-apprapp",
+                "version": "v1.5.0",
+                "optional": true,
+                "input_mapping": [
+                  {
+                    "dependency_output": "region",
```

Comment:
typically the parent decides the region and we pass it to the dependency

Reply:
right, the Workload protection should be passing its region to App config DA

Reply:
Can we double check that both services support the same regions? If the don't we may need to expose another virtual input just for the AppConfig region